docs/OnnxReleases.md

```diff
@@ -96,7 +96,7 @@ The ONNX project, going forward, will plan to release roughly on a two month cad
 **Source Distribution**
 * Follow the same process in TestPyPI to produce the source distribution.
 * Use ``twine upload --verbose dist/* --repository-url https://upload.pypi.org/legacy/`` instead to upload to the official PyPI.
-* Test with ``pip install --no-binary :all: onnx``
+* Test with ``pip install --no-binary onnx onnx``
```

Comment:
what does this change imply?

Reply:
Previously --no-binary :all: means all binary packages (including dependencies) are disabled, but actually ONNX should only disable "onnx" binary here (only build ONNX from source distribution).